tests/util/test_vamp_integration.cpp

```diff
@@ -28,44 +24,6 @@ BOOST_AUTO_TEST_CASE(VampTargetLinkingTest)
     BOOST_CHECK_MESSAGE(true, "vamp_cpp target successfully linked");
 }
 
-// Test OMPL to VAMP state conversion for Panda robot
```

Comment:
are you planning to have some basic test code for your own ompl-vamp integration?